packages/app/src/app/index.html

```diff
@@ -43,6 +43,7 @@
   <link crossOrigin="anonymous" rel="preload" href="/static/fonts/inter/Inter-Bold.woff2" as="font">
   </link>
   <link href="/static/fonts/inter/inter.css" rel="stylesheet">
+  <link href="https://www.monolisa.dev/static/fonts.css" rel="stylesheet">
```

Comment:
Mistake, remove tomorrow sara

Reply:
We have a smaller CSS file. @bebraw what's the name?